ee/api/test/test_funnel_correlation.py

```diff
@@ -0,0 +1,70 @@
+from typing import Any, Dict
+
+import pytest
+from django.test import Client
+from freezegun import freeze_time
+
+from posthog.test.base import BaseTest
+
+
+@pytest.mark.clickhouse_only
+class FunnelCorrelationTest(BaseTest):
+    """
+    TODO: fill in details of request structure. At the moment it's not needed as
+    we just return mock data
+    """
+
+    def test_requires_authn(self):
+        response = get_funnel_correlation(client=self.client, team_id=self.team.pk,)
+        assert response.status_code == 401
+
+    def test_event_correlation_endpoint(self):
+        with freeze_time("2020-01-01"):
+            self.client.force_login(self.user)
+
+            odds = get_funnel_correlation_ok(client=self.client, team_id=self.team.pk,)
+
+        self.assertEqual(
+            odds,
+            {
+                "is_cached": False,
+                "last_refresh": "2020-01-01T00:00:00Z",
+                "result": {
+                    "events": [
+                        # Top 10
+                        {"event": "Event A", "success": 1, "failure": 4, "odds_ratio": 10},
+                        {"event": "Event B", "success": 1, "failure": 4, "odds_ratio": 9.4},
+                        {"event": "Event C", "success": 1, "failure": 4, "odds_ratio": 5.4},
+                        {"event": "Event D", "success": 1, "failure": 4, "odds_ratio": 3.4},
+                        {"event": "Event E", "success": 1, "failure": 4, "odds_ratio": 2.6},
+                        {"event": "Event F", "success": 1, "failure": 4, "odds_ratio": 2.5},
+                        {"event": "Event G", "success": 1, "failure": 4, "odds_ratio": 2.1},
+                        {"event": "Event H", "success": 1, "failure": 4, "odds_ratio": 2.0},
+                        {"event": "Event I", "success": 1, "failure": 4, "odds_ratio": 1.5},
+                        {"event": "Event J", "success": 1, "failure": 4, "odds_ratio": 1.2},
+                        # Bottom 10
```

Comment:
It's fine with just the top 10 right now? You don't have to try and adjust the code to get top and bottom 10 out, as I'm pretty sure we'll be changing the structure for odds of success/failure - to return only one of these at a time.